content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -332,7 +339,7 @@ In addition to Kubernetes pod and Kubernetes node tags, the following out-of-the
 | `persistent_volume_reclaim_policy`      | The Kubernetes Reclaim Policy on the Persistent Volume.                                                                                      |
 | `storage_class_name`                    | The Kubernetes Storage Class used to instantiate the Persistent Volume.                                                                      |
 | `volume_mode`                           | The Volume Mode of the Persistent Volume.                                                                                                    |
-| `ebs_volume_type`                       | The type of the AWS EBS volume. Can be `gp3`, `gp2`, or others.                                                                              |
+| `ebs_volume_type`                       | The type of the EBS volume. Can be `gp3`, `gp2`, or others.                                                                              |
```

Comment:
Any reason you removed the AWS here?

Reply:
was failing the vale check, reason was it did not approve of having 'AWS EBS' so maybe new linter rules were added?